aws-core/src/main/java/com/amplifyframework/auth/AWSCredentialsProvider.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Date-free copyright message, or 

```suggestion
 * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```